.circleci/config.yml

```diff
@@ -7,13 +7,16 @@ workflows:
   build:
     jobs:
       - elixir/build_test:
+          cache-version: 4
           filters: &filters
             tags:
               only: /v.*/
       - elixir/test:
+          cache-version: 4
           filters:
             <<: *filters
       - elixir/lint:
+          cache-version: 4
           filters:
             <<: *filters
       - elixir/hex_publish:
```

Comment:
let's add `cache-version` to the `hex_publish` as well, otherwise it might fail while publishing